tests/test_cli.py

```diff
@@ -327,3 +331,77 @@ def test_multiple_folders(script_runner):
+    assert bboxList == pytest.approx([96.21146, 25.55834, 96.35495, 25.63293], abs=tolerance)
+    assert "4326" in result
+
+
+def test_zenodo_valid_doi_repository(script_runner):
+    ret = script_runner.run('geoextent',
+                            '-b', '-t', 'https://doi.org/10.5281/zenodo.820562')
+    assert ret.success, "process should return success"
+    assert 'has no identifiable time extent' in ret.stderr
+    result = ret.stdout
+    bboxList = parse_coordinates(result)
+    assert bboxList == pytest.approx([96.21146, 25.55834, 96.35495, 25.63293], abs=tolerance)
+    assert "4326" in result
+
+
+def test_zenodo_valid_link_repository_with_no_geoextent(script_runner):
+    ret = script_runner.run('geoextent', '-b', '-t', 'https://zenodo.org/record/1810558')
+    result = ret.stdout
+    assert "bbox" not in result, "This repository contains a PDF file, it should not return a bbox"
+    assert "tbox" not in result, "This repository contains a PDF file, it should not return a tbox"
+
+
+def test_zenodo_invalid_link_repository(script_runner):
+    ret = script_runner.run('geoextent',
+                            '-b', '-t', 'https://zenado.org/record/820562')
+    assert not ret.success, 'Typo in URL'
+    assert "is not a valid" in ret.stderr, 'Typo in URL'
+
+
+def test_zenodo_valid_but_removed_repository(script_runner):
+    ret = script_runner.run('geoextent', '-b', '-t', 'https://zenodo.org/record/1')
+    assert not ret.success
+    assert "does not exist" in ret.stderr
+
+
+def test_zenodo_valid_but_no_extraction_options(script_runner):
+    ret = script_runner.run('geoextent', 'https://zenodo.org/record/1')
+    assert not ret.success, 'No extractions options, geoextent should fail'
+    assert "Require at least one of extraction options, but bbox is False and tbox is False" in ret.stderr
+
+
+def test_zenodo_valid_but_not_open_access(script_runner):
+    ret = script_runner.run('geoextent', '-b', '-t', 'https://zenodo.org/record/51746')
+    assert not ret.success, 'The repository exists but it is not accessible. Geoextent should fail'
+    assert "This record does not have Open Access files. Verify the Access rights of the record" in ret.stderr
+
+
+def test_export(script_runner):
+    with tempfile.TemporaryDirectory() as tmp:
+        gpkg_file = os.path.join(tmp, "export_file.gpkg")
+        script_runner.run('geoextent', '-b', '-t', '--output', gpkg_file, 'tests/testdata/folders/folder_two_files')
+        assert os.path.exists(gpkg_file)
+        files_gdf = gpd.read_file(gpkg_file, layer="files")
+        geo_version = "geoextent:" + current_version
+        output = files_gdf.loc[lambda df: files_gdf['handler'] == geo_version, ]
+        tbox = list(output['tbox'])
+        bounds = output.bounds
+        bbox = list(bounds.iloc[0])
+    assert tbox[0] == "2018-11-14/2019-09-11"
+    #assert bbox == pytest.approx([2.05233, 41.31703, 7.64725, 51.97462], abs=tolerance)
+
+
+def test_export_no_output_file(script_runner):
```

Comment:
Could you also have a different order here, and test that?

```
('geoextent', '-b', '--output', '-t', 'tests/testdata/folders/folder_two_files')
```

Reply:
This should fail 
`error: argument --output: expected one argument`